code/modules/spells/aoe_turf/conjure/forcewall.dm

```diff
@@ -69,3 +104,36 @@
 	new /obj/effect/forcefield/cult(get_turf(src))
 	qdel(src)
 	return
+
+/*
+Unwall fields
+-kanef
+*/
+/obj/effect/unwall_field
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "fuel"
+	name = "invisible un-wall"
+	desc = "You have a REALLY bad feeling about this."
+	anchored = 1.0
+	opacity = 0
+	var/duration = 300 // How long the wall lasts, in ticks
+	var/static/list/forbidden_passes = list(/turf/unsimulated/wall,/turf/simulated/wall/invulnerable,/obj/structure/grille/invulnerable) // To stop people breaking maps like centcomm or lamprey stuff
+
+/obj/effect/unwall_field/permanent // For future mapping or bus shenanigans
+	duration = 0 // Forever
+
+/obj/effect/unwall_field/New()
+	..()
+	if(duration) // Wait the duration if any and delete it, if zero, don't
+		spawn(duration)
+			qdel(src)
+
+/obj/effect/unwall_field/to_bump(atom/A)
+	if(ismovable(A)) // For forceMove
+		var/atom/movable/AM = A
```

Comment:
Why not just type it as `/atom/movable/A` to begin with? It's always going to be one anyway.

Reply:
Fixed